db/fixtures/ae_datastore/Deployment/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/ssh'
+require 'rest_client'
+
+def provider_token
+  token = ""
+  Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
```

Comment:
@dkorn is it ok not to check the host key? (in this all all the executions below)